tools/release.js

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env node
+'use strict';
+
+const yargs = require('yargs/yargs');
+const { hideBin } = require('yargs/helpers');
+const readline = require('readline').createInterface({
```

Comment:
I prefer [`prompts`](https://www.npmjs.com/package/prompts) over `readline`, but no biggie of course

Reply:
I think this is fine. I tried to keep the dependencies to a minimum, although I guess it doesn't really matter since the jump from zero to one dependency is way greater than the jump from 3 to 4